src/base/bittorrent/private/filterparserthread.cpp

```diff
@@ -444,3 +471,42 @@ void FilterParserThread::run()
 
     qDebug("IP Filter thread: finished parsing, filter applied");
 }
+
+QVector<int> FilterParserThread::indicesOfDelimiters(const char * const data, const char delimiter, const int start, const int end) const
```

Comment:
I am not what you mean exactly...
The only memory allocations I see for this function is the std::vector and the ints that hold the indices.